Mark_III_WB_Search/FakeDataBase.mm

```diff
@@ -14,8 +14,12 @@
 
 // For the simplicity of the task we assume that this function always returns 10000 items
 Event * __strong * fetchEvents(NSDate *startDate, NSDate *endDate) {
-    Event * __strong *eventsBuffer = (Event * __strong *)malloc(sizeof(Event *) * 10000);
```

Comment:
This is fixable by replacing `malloc` with `calloc`, no need in the next 3 lines

Reply:
I agree!